browser-based-ingestion/README.md

```diff
@@ -0,0 +1,249 @@
+### KVS Browser based Ingestion is used to stream video from the system's webcam to the KVS backend directly without any proxies. ###
+
+Currently, it is only available via Chrome. 
+There are four important tasks that this repository helps perform
+1. Obtaining the video from the webcam/ uploading video file
+2. Converting it to containers and codecs expected by the playback
+3. Sending to the putMedia API via POST calls
+4. Viewing the stream on the browser
+
+
+### Setup ###
```

Comment:
I think you just want people do `git clone` this repo? I should just have to do `npm install` and then `npm run server` or something like that.

Reply:
Yes, now, I have done that. I have provided all code in one place. All they would have to do is include scripts and import functions after cloning the repository